frontend/pages/admin/UserManagementPage/components/UsersTable/UsersTable.tsx

```diff
@@ -251,6 +263,18 @@ const UsersTable = ({ router }: IUsersTableProps): JSX.Element => {
             setCreateUserErrors({
               password: "Password must meet the criteria below",
             });
+          } else if (
+            userErrors.data.errors?.[0].reason.includes("password too long")
+          ) {
+            renderFlash(
+              "error",
+              <p>
+                Could not create user. Password is over the 48 characters limit.
+                If the password is under 48 characters,
+                <br /> please check the auth_salt_key_size in your Fleet server
+                config.
+              </p>
+            );
```

Comment:
any thoughts on having a helper function or a constant somewhere at least with the html bits? seems like at least on this file would be nice to have

Reply:
Refactored this per Noah's UI changes, so no need :D